ksqldb-version-metrics-client/pom.xml

```diff
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>org.mock-server</groupId>
             <artifactId>mockserver-netty</artifactId>
-            <version>3.10.4</version>
+            <version>5.11.2</version>
```

Comment:
Why we we set this version explicitly and not inherit from top level pom?

Also, as we do set the version explicitly, how could it happen that the build broke? Seems we pulled in a different version somehow during runtime? Is there something wrong with our setup?

Reply:
My understanding is that another library from common, netty, overrode the version relied by on this library, to be a newer incompatible version. I effectively am upping the version of this library to now allow for a newer version of netty.